s8/array_meth.rb

```diff
@@ -0,0 +1,48 @@
+# .new
+p Array.new
+p Array.new.class
+
+p Array.new(3)
+p Array.new(3, "a")
+p Array.new(3, [])
+puts
+
+# access with index
+a = %w(a b c d e f g)
+p a.length
```

Comment:
What's the difference between `length` and `size`?

Reply:
According to the course, these do not have a difference but consulting I read something about that, length is not part of the Enumerable method, so this could offer a faster result, despite being designed for Strings and Arrays.